### DISCUSSION TITLE
OpenCV 中那些深度學習模組

### DISCUSSION THREAD

**Discussion by minniewei** [1101-GS4719-C/Python-Community-GS4719-C#495] [General]
很多時候我們會使用open-cv來進行視覺辨識相關處理。以下這篇文章講述了我們常用到的幾個模組，希望能夠給看得懂的人帶來幫助！！（我是有點看不懂啦......)
https://www.gushiciku.cn/pl/2Z9v/zh-tw

**Comment by ghost** [comment]
#412 (之前自己也有試試一些功能，的確更深入的需要花時間去了解w)
都可以稍微用一些圖來試試看，畢竟這是機器學習影像辨識算最基礎也最能由淺入深的模組

**Comment by ThomasMeng0515** [comment]
YouTube上有基礎的OpenCV實際操作教學，之前有稍微看過感覺很不錯，給大家參考~
https://youtu.be/xjrykYpaBBM

**Comment by xkllkx** [comment]
如果不介意印度口音的英文，YT上也有許多印度人的教學影片，覺得很受用。

**Comment by jason79461385** [comment]
我之前看朋友做專題有用到透過色差的方式來判別木材的節，但要調整蠻多參數，可能背景變了就辨識不到
應該還是要透過機器學習的方式來輔助影像辨識，但在train的時候要先準備超多張照片

**Comment by awkward-willy** [comment]
先前看 Youtube 上有許多相關的教學
覺得蠻有趣的
不過自己也沒有時間深入研究
希望自己未來閒暇時間能在這方面有更多的著墨

**Comment by treeann** [comment]
OPencv感覺超厲害ㄝ  我在深度學習專題時也有使用到

**Comment by liangyinglly** [comment]
看不懂的話應該可以修圖像辨識那堂通識吧

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by Lynn12378** [comment]
感謝分享，雖然我看不懂QQ希望之後有機會可以了解更多深度學習

**Comment by Jimmy911218** [comment]
謝謝樓主提供
